src/screens/ExchangeModal.tsx

```diff
@@ -1280,6 +1280,9 @@ export default function ExchangeModal({
                 marginBottom={0}
                 marginTop={0}
                 testID={`${testID}-gas`}
+                crossChainServiceTime={
+                  (tradeDetails as CrosschainQuote)?.routes?.[0]?.serviceTime
```

Comment:
do you want to create a helper to get this param maybe like https://github.com/rainbow-me/rainbow/blob/develop/src/handlers/swap.ts#L101 

i see this being used in different places

Reply:
yeah agree